docs/TektonConfig.md

```diff
@@ -178,6 +178,55 @@ pipeline:
     kube-api-burst: 10
 ```
 
+
+### Chain
+
+Chain section allows user to customize the Tekton Chain features. This allows user to customize the values in configmaps
+
+Refer to [properties](https://github.com/tektoncd/operator/blob/main/docs/TektonChain.md#chain-config) section in TektonChain for available options
+
+Example:
+
+```yaml
+chain:
+  targetNamespace: tekton-pipelines
+  controllerEnvs:
+    - name: MONGO_SERVER_URL      # This field is optional and when added by user, it is added as env to Chains controller
```

Comment:
we need to mention that this is the only key supported for the moment

Reply:
yes 👍🏻